### PR TITLE
Add functions to get games user follows

### PR DESCRIPTION
**Pull request by scottnuma**

### CODE REVIEW

File: user_taste.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from twitch import TwitchClient
+
+
+def get_all_follows(user_id: str, show_progress=False):
+    """Return a list of all channels a user follows"""
```
Comment (kcempron) [inline user_taste.py]: Can we have the same description format as the one below:
"""
Description
"""

**Comment by kcempron** [comment]
Also, have we tested this?
I know it might be tedious for a class project but might be good to write in some unit tests :)

**Comment by scottnuma** [comment]
I've done local testing to confirm.

I suppose networked unit tests are a reasonable step? I haven't done much
with network spoofs in Python.

**Comment by kcempron** [comment]
@scottnuma as long as it's tested, it's fine
don't need to add unnecessary work for ourselves

**Pull request merged**